src/components/pages/RequestView.vue

```diff
@@ -0,0 +1,36 @@
+<script>
+export default {
+  props: {
+    active: {
+      type: Boolean,
+      required: true,
+    },
+  },
+};
+</script>
+
+<template>
+  <div>
+    <div id="titlebar" class="single">
+      <div class="container">
+        <div class="sixteen columns">
+          <h2>My Account</h2>
+        </div>
+      </div>
+    </div>
+    <div class="my-account">
+      <ul class="tabs-nav">
+        <li :class="{'active': active}">
```

Comment:
`:class` icine yazdigimiz object'in oncesinde ve sonrasinda bir bosluk birakiyoruz.

`<li :class="{ 'active': active }">`

yukardaki sekilde daha okunabilir.

Ayrica cok buyuk ihtimalle object icindeki key olan `active`'in string olmasina gerek yok.

`<li :class="{ active: active }">`

Hatta muhtemelen calisacaktir ama calismazsa da sasirmayacagim sunu da dener misiniz?

`<li :class="{ active }">`

Reply:
Bu şekilde çalıştı, çok da mantıklı 

`<li :class="{  active  }">`

Ama alt bu şekilde çalışmadı 

`<li :class="{  !active  }">`

şöyle bıraktım

`<li :class="{  active:  !active  }">`